client/js/keybinds.ts

```diff
@@ -83,6 +83,35 @@ Mousetrap.bind(["alt+shift+up", "alt+shift+down"], function (e, keys) {
 	return false;
 });
 
+// Switch to the next/previous unread chat
+Mousetrap.bind(["alt+mod+up", "alt+mod+down"], function (e, keys) {
```

Comment:
hm, on a second thought... I assume "mod" is the windows key?
Can we leave that one alone and use Ctrl+Alt+{up,down} ?

Windows is frequently already used by the window manager and executes global hotkeys rather than local ones.

Would also make it symmetrical with the rest

* Alt+key == window
* Alt+shift+key == lobby
* Alt+ctrl+key == activity

Bad idea? What made you pick mod?

Reply:
windows key would be `meta`

`mod` is `ctrl` on windows and `cmd` on osx. 
so on windows this is `alt+ctrl+up/down` and on mac this is `opt+cmd+up/down`

Reply:
ack, thanks for the explanation